src/main/controllers/RemoveListConfirmationController.ts

```diff
@@ -14,44 +14,37 @@ const userManagementService = new UserManagementService();
 
 export default class RemoveListConfirmationController {
     public async get(req: PipRequest, res: Response): Promise<void> {
-        const artefactId = req.query.artefact;
-        if (artefactId) {
-            const artefact = await publicationService.getIndividualPublicationMetadata(
-                artefactId,
-                req.user['userId'],
-                true
-            );
-            artefact.listTypeName = manualUploadService.getListItemName(artefact.listType);
-            artefact.contDate = DateTime.fromISO(artefact.contentDate, {
-                zone: 'Europe/London',
-            }).toFormat('dd MMMM yyyy');
-            res.render('remove-list-confirmation', {
-                ...cloneDeep(req.i18n.getDataByLanguage(req.lng)['remove-list-confirmation']),
-                artefact,
-                court: await courtService.getLocationById(artefact.locationId),
-                displayError: false,
-            });
-        } else {
-            res.render('error', req.i18n.getDataByLanguage(req.lng).error);
+        const formData = req.cookies?.formCookie ? JSON.parse(req.cookies['formCookie']) : {};
+        const listsToDelete = RemoveListConfirmationController.getSelectedLists(formData);
+        const listData = [];
+        for (const list of listsToDelete) {
+            await addListDetailsToArray(list, req.user?.['userId'], listData);
         }
+        res.render('remove-list-confirmation', {
+            ...cloneDeep(req.i18n.getDataByLanguage(req.lng)['remove-list-confirmation']),
+            removalList: manualUploadService.formatListRemovalValues(listData),
+            locationId: formData.locationId,
+            court: await courtService.getLocationById(formData.locationId),
+            displayError: false,
+        });
     }
 
     public async post(req: PipRequest, res: Response): Promise<void> {
```

Comment:
There should be a check in this method to make sure req.body, req.body.artefactIds and req.body.locationId's exists.

Unlikely scenario but I think it will cause a crash if they're not there